INFO:app:static
INFO:app:static
INFO:app:static
INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8100
 * Running on http://172.16.5.4:8100
INFO:werkzeug:[33mPress CTRL+C to quit[0m
INFO:app:static
INFO:app:static
INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8001
 * Running on http://172.16.5.4:8001
INFO:werkzeug:[33mPress CTRL+C to quit[0m
INFO:werkzeug:127.0.0.1 - - [14/Apr/2024 08:51:27] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
INFO:app:calling download
ERROR:app:Exception on / [GET]
Traceback (most recent call last):
  File "/home/codespace/.python/current/lib/python3.10/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/codespace/.python/current/lib/python3.10/site-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/codespace/.python/current/lib/python3.10/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/codespace/.python/current/lib/python3.10/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/workspaces/Assignment2/clo835/app.py", line 60, in main
    return render_template('home.html', debug="Environment Variables: DB_Host=" + (os.environ.get('MYSQL_SERVICE_HOST') or "Not Set") + "; DB_Database=" + (os.environ.get('DB_Database') or "Not Set") + "; DB_User=" + (os.environ.get('DB_User') or "Not Set") + "; DB_Password=" + (os.environ.get('DB_Password') or "Not Set") + "; " + err_message, db_connect_result=db_connect_result, name=socket.gethostname(), color=color, image_url=IMAGE_URL, user_name=USER_NAME)
  File "/home/codespace/.python/current/lib/python3.10/site-packages/flask/templating.py", line 148, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/home/codespace/.local/lib/python3.10/site-packages/jinja2/environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/home/codespace/.local/lib/python3.10/site-packages/jinja2/environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "/home/codespace/.local/lib/python3.10/site-packages/jinja2/environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "/home/codespace/.local/lib/python3.10/site-packages/jinja2/loaders.py", line 125, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/home/codespace/.python/current/lib/python3.10/site-packages/flask/templating.py", line 59, in get_source
    return self._get_source_fast(environment, template)
  File "/home/codespace/.python/current/lib/python3.10/site-packages/flask/templating.py", line 95, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: home.html
INFO:werkzeug:127.0.0.1 - - [14/Apr/2024 08:51:28] "[35m[1mGET / HTTP/1.1[0m" 500 -
INFO:app:static
INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8001
 * Running on http://172.16.5.4:8001
INFO:werkzeug:[33mPress CTRL+C to quit[0m
INFO:werkzeug: * Restarting with stat
INFO:app:static
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 571-981-926
INFO:app:calling download
INFO:werkzeug:127.0.0.1 - - [14/Apr/2024 08:51:57] "[35m[1mGET / HTTP/1.1[0m" 500 -
INFO:werkzeug:127.0.0.1 - - [14/Apr/2024 08:51:57] "GET /?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [14/Apr/2024 08:51:57] "GET /?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [14/Apr/2024 08:51:57] "GET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [14/Apr/2024 08:51:57] "[36mGET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 -
INFO:werkzeug: * Detected change in '/workspaces/Assignment2/clo835/app.py', reloading
INFO:werkzeug: * Restarting with stat
INFO:werkzeug: * Restarting with stat
